src/librbd/cache/SharedPersistentObjectCacher.h

```diff
@@ -0,0 +1,42 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: can this be combined w/ the parent image cache object dispatch below?

Reply:
done. this class became internal class of Parent Cache Object dispatch. thanks